src/wp-admin/customize.php

```diff
@@ -29,7 +29,7 @@
 if ( $wp_customize->changeset_post_id() ) {
 	$changeset_post = get_post( $wp_customize->changeset_post_id() );
 
-	if ( ! current_user_can( get_post_type_object( 'customize_changeset' )->cap->edit_post, $changeset_post->ID ) ) {
+	if ( ! current_user_can( 'edit_post', $changeset_post->ID ) ) {
```

Comment:
This doesn't seem to work because `customize_changeset` is using another meta cap `customize` for `edit_post`.

Probably related to https://core.trac.wordpress.org/ticket/40922.